frontend/src/pages/auth/Login/hooks/useLogin.js

```diff
@@ -6,10 +7,14 @@ import authApi from "../../../../api/authApi";
 export const useLogin = () => {
   const navigate = useNavigate();
 
+  const [loading, setLoading] = useState(false);
```

Comment:
- i think this kind of states e.g. api related better to store in the reducer. we access it nalang via reducer (Apply to all related)